### DISCUSSION TITLE
[popup] Open links in Classic when use it in Classic

### DISCUSSION THREAD

**Discussion by shellma1211** [tprouvot/Salesforce-Inspector-reloaded#620] [Ideas]
It will be great if when I use Salesforce inspector in Classic(for example, open object page, field page), it will open the page in Classic. If I use it in Lightning, it will open links in lightning.
![image](https://github.com/user-attachments/assets/49c7015a-fbab-45da-a42e-86c14a89dfcb)

**Comment by tprouvot** [comment]
Hi @shellma1211,

Thanks for creating the idea but I'm not very comfortable with developing something related to classic since this is not maintained anymore 🫤
